server/src/main/java/org/elasticsearch/search/aggregations/bucket/range/RangeAggregator.java

```diff
@@ -434,6 +434,7 @@ public static Aggregator buildWithoutAttemptedToAdaptToFilters(
     private final DocValueFormat format;
     protected final Range[] ranges;
     private final boolean keyed;
+    @SuppressWarnings("rawtypes")
```

Comment:
Why is it we don't just use wildcard types in these scenarios?

Reply:
You get into the situation of `?` being different from `?`. It's a mess.